containers/flatpak/cockpit-ssh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "$@" >&2
+
+flatpak-spawn --host ssh "$1" cockpit-bridge
```

Comment:
Likewise, why do we need this wrapper? cockpit-ws can be attached directly to "ssh cockpit-bridge" like c-desktop does?

Reply:
This allows choosing the host via the UI in the browser instead of on the commandline.  That's the major change here, and I think it's a valuable one.

We probably don't need this wrapper, though: we could just as easily install a custom ssh manifest which knows how to call `flatpak-spawn`.  I also started writing a python wrapper which knows how to do some rudimentary authentication handling (the password field, for example, is currently completely useless), but this is going to be some black magic, if it ever works at all.